keycloak/keycloak-operator
# User CRD

User CR

Verification steps:

1. Deploy Keycloak and a Realm
2. Create the example users
3. Play with their roles and make sure the updates are reflected in the Admin console
